### ISSUE TITLE
Multiple Native transformations in a transform statement

### ISSUE THREAD

**Issue by CoolsJoris** [Seddryck/NBi#513]
Hi, 
It seems that i can't chain native transformations inside a transform statement.
What i would like to do (for example):
```xml
<transform language="native" original-type="text" column="[ColumnName]">
  empty-to-null | null-to-zero
</transform>
```

The current workaround that is working:
```xml
<transform language="native" original-type="text" column="[ColumnName]">empty-to-null</transform>
<transform language="native" original-type="text" column="[ColumnName]">null-to-zero</transform>
```

Is this a possibility to add this feature ?

Thank you.

**Comment by Seddryck** [comment]
Yes for sure, will be added in the next release. I'll also include a small enhancement to avoid some silly error messages when starting a transformation by `value` ... to be aligned with `c-sharp` syntax.

**Comment by Seddryck** [comment]
Beta available in `1.21.0-beta0332` or at https://ci.appveyor.com/project/Seddryck/nbi/builds/27252363/artifacts
